src/main/kotlin/io/github/anki/anki/service/model/Deck.kt

```diff
@@ -1,8 +1,8 @@
 package io.github.anki.anki.service.model
 
 data class Deck(
-    var id: String? = null,
-    var userId: String,
-    var name: String,
-    var description: String?,
+    val id: String? = null,
+    val userId: String,
+    val name: String?,
```

Comment:
maybe set default `val name: String?  = null`?

Reply:
Deck has to have name